fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/PrivEntry.java

```diff
@@ -187,4 +188,29 @@ protected static int compareAssist(String... strings) {
         }
         return 0;
     }
+
+    protected PrivEntry copy() throws AnalysisException, PatternMatcherException {
```

Comment:
change this method to `abstract`,then write the `copy logic` to each subclass

Reply:
changed, thank you for your review.